lib/coins/ada.js

```diff
@@ -0,0 +1,50 @@
+const BASE58 = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
```

Comment:
It seems you miss `I` here, is not there a better way to create that? Using some JS methods returning chars for ascii positions?

Reply:
> Compared with Base64, the following similar-looking letters are omitted: 0 (zero), O (capital o), I (capital i) and l (lower case L) as well as the non-alphanumeric characters + (plus) and / (slash).

Reply:
sorry, my bad